x/merkledb/node.go

```diff
@@ -173,6 +195,8 @@ func (n *node) clone() *node {
 			children: maps.Clone(n.children),
 		},
 		valueDigest: n.valueDigest,
+		size:        n.size,
+		nodeBytes:   n.nodeBytes,
```

Comment:
nodeBytes is only read/written in node.Clone and during db.commitChanges.  Clone only happens during NewView, which takes the commit read lock, so cannot occur at the same time as db.commitChanges, which takes the commit write lock.

Reply:
also in getEditable node, which takes the db read lock, but commitchanges takes the db write lock, so that is also fine.

Reply:
^ Can we put this in a comment?